lib/datadog/opentelemetry/sdk/trace/span.rb

```diff
@@ -15,6 +15,20 @@ def set_attribute(key, value)
           res
         end
 
+        def record_exception(exception, attributes: nil)
+          res = super
+          if (span = datadog_span)
+            span.set_error(
+              [
+                attributes&.fetch('exception.type', nil) || exception.class.to_s,
```

Comment:
If you mean to get the class name, `exception.class.name` ?

Reply:
I am using the same convention as the otel sdk: https://github.com/open-telemetry/opentelemetry-ruby/blob/35abeac8653359aebd0231c94a23587ad4ee372e/sdk/lib/opentelemetry/sdk/trace/span.rb#L166. I am not sure why they used ` exception.class.to_s` but I think it's best to keep things consistent.